### ISSUE TITLE
Implementing a `contrastRatio` method

### ISSUE THREAD

**Issue by yannickl** [yannickl/DynamicColor#21]
I should be great to implement a `contrastRatio` method following the [WCAG 2.0](https://www.w3.org/TR/WCAG20-TECHS/G18.html)  recommendation.

It is related to the #15 and #23 implementation.

**Comment by Onejjy** [comment]
If it's Ok for you, I'll take it (if nobody else want it :p ).

**Comment by yannickl** [comment]
First-come, first-served basis here. So go ahead, have fun! ;)

**Comment by yannickl** [comment]
Thank you @Onejjy for the implementation! I close the issue.
